metadata-ingestion/setup.py

```diff
@@ -90,9 +90,9 @@ def get_long_description():
     # Required for all SQL sources.
     "sqlalchemy==1.3.24",
     # Required for SQL profiling.
-    "great-expectations>=0.14.11,<0.15.3",
-    # datahub does not depend on Jinja2 directly but great expectations does. With Jinja2 3.1.0 GE 0.14.11 is breaking
-    "Jinja2<3.1.0",
+    "great-expectations>=0.15.12",
```

Comment:
great-expectations actually dropped support for Python 3.6, and so requiring `>=0.15.12` isn't compatible with DataHub at the moment.

However, we're also working on dropping support for Python 3.6 (see https://github.com/datahub-project/datahub/pull/5521), so this won't be an issue for too much longer.

We've got two options then:
1. Wait until datahub drops 3.6 and then merge this PR without issue
2. Modify this PR to use a GE release that is compatible with 3.6

I think the former option makes more sense, but I know that this PR has been sitting for a while and it's annoying to wait even more.

Reply:
Hi! Thanks for taking up this PR. I agree that the former is a better choice here. Lets come back to this when datahub drops 3.6 .

This PR is highly dependent on the fix i made in this release for GE [here](https://github.com/great-expectations/great_expectations/releases/tag/0.15.11) which is at least 0.15.11 .